demisto_sdk/commands/validate/tests/BA_validators_test.py

```diff
@@ -1018,3 +1021,66 @@ def test_IsValidVersionValidator_fix():
         == "Updated the content item version to -1."
     )
     assert content_item.version == -1
+
+
+@pytest.mark.parametrize(
+    "content_items, expected_msg",
+    [
+        pytest.param(
+            [
+                create_integration_object(
+                    paths=["name", "display"],
+                    values=["Test v1", "Testv1"],
+                ),
+                create_integration_object(
+                    paths=["name", "display"],
+                    values=["Test Integration", "TestIntegration"],
+                ),
+            ],
+            "The following fields: name, display shouldn't contain the word 'Integration'.",
+            id="Integration Test (valid and invalid)",
+        ),
+        pytest.param(
+            [
+                create_script_object(
+                    paths=["name", "display"],
+                    values=["Test v1", "Testv1"],
+                ),
+                create_script_object(
+                    paths=["name", "display"],
+                    values=["Test Script", "TestScript"],
+                ),
+            ],
+            "The following fields: name, display shouldn't contain the word 'Script'.",
+            id="Script Test (valid and invalid)",
+        ),
+        pytest.param(
+            [
+                create_playbook_object(
+                    paths=["name", "display"],
+                    values=["Test v1", "Testv1"],
+                ),
+                create_playbook_object(
+                    paths=["name", "display"],
+                    values=["Test Playbook", "TestPlaybook"],
+                ),
+            ],
+            "The following fields: name, display shouldn't contain the word 'Playbook'.",
+            id="Playbook Test (valid and invalid)",
+        ),
+    ],
+)
+def test_IsEntityTypeInEntityNameValidator_is_valid(content_items, expected_msg):
+    """
+    Given
+    - Case 1: A content item doest not have its type in name of display felids.
```

Comment:
Isn't there 3 tests with 2 content items each?
Let's change to something like 
- 2 integrations
    - One valid integration with ...
    - One invalid integration with...

Reply:
Done.